lib/open_api_spex/schema.ex

```diff
@@ -250,6 +251,7 @@ defmodule OpenApiSpex.Schema do
           xml: Xml.t() | nil,
           externalDocs: ExternalDocumentation.t() | nil,
           example: any,
+          examples: [any],
```

Comment:
```suggestion
          examples: [any] | nil,
```

Reply:
Ah, indeed. I just pushed the change you mentioned.